[WIP] feat: prepare events for sending collectibles

briansztamfater opened the pull request:
fixes #18260 

### Summary

This PR adds basic events to send a collectible from our wallet

#### Platforms

- Android
- iOS

#### Areas that maybe impacted

##### Functional

- wallet / transactions

### Steps to test

_Will update steps when PR is ready_

status: wip

briansztamfater committed:
feat: prepare events for sending collectible

Signed-off-by: Brian Sztamfater <brian@status.im>

status-im-auto commented:
### Jenkins Builds
| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :x: | 947b8e1a | [#2](https://ci.status.im/job/status-mobile/job/prs/job/tests/job/PR-18361/2/) | 2024-01-02 16:23:49 | ~2 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-mobile/job/prs/job/tests/job/PR-18361/2/consoleText) |

ulisesmac committed:
Code style

ulisesmac committed:
Show collectibles to send and add filter capabilities

Additionally,
- Replace the sub `:wallet/collectibles-per-account` by `:wallet/current-viewing-account-collectibles` since
  it was only used for the current viewing account and simplified the code.
- Refactor the select-asset view.

ulisesmac committed:
Generalize collectibles-tab view

ulisesmac committed:
Lint fix

ulisesmac committed:
Add translations for not found state

ulisesmac committed:
Add empty state image

ulisesmac committed:
Lint fix

briansztamfater commented:
Continue in #18473
